styles/designs/webview/parts/_exercise-components.scss

```diff
@@ -181,54 +181,6 @@ $Exercise__Solution__Container--WithFirstElement: (
     )
 );
 
-$Exercise--ProblemsExercises__Problem--Before1: (
```

Comment:
these components are only used under the `problems-exercise` selectors, which don't appear in the content.

additionally, they inject `Problem` and `Solution` titles via the styles, which is no longer the preferred way (preferred way: cookbook, using I18n for translation support)